jnigen/lib/src/config/yaml_reader.dart

```diff
@@ -42,19 +44,15 @@ class YamlReader {
       stderr.writeln(parser.usage);
       exit(1);
     }
-    final configFile = results['config'] as String?;
-    Map<dynamic, dynamic> yamlMap = {};
-    if (configFile != null) {
+    final configFilePath = results['config'] as String?;
+    String? configFileContents;
+    Uri? configFileUri;
+    if (configFilePath != null) {
       try {
-        final yamlInput = loadYaml(File(configFile).readAsStringSync(),
-            sourceUrl: Uri.file(configFile));
-        if (yamlInput is Map) {
-          yamlMap = yamlInput;
-        } else {
-          throw ConfigException('YAML config must be set of key value pairs');
-        }
+        configFileContents = File(configFilePath).readAsStringSync();
+        configFileUri = File(configFilePath).uri;
       } on Exception catch (e) {
-        stderr.writeln('cannot read $configFile: $e');
+        stderr.writeln('Cannot read $configFilePath: $e.');
```

Comment:
Shouldn't this be a fatal error?

Reply:
Maybe, I tried to keep this PR behavior preserving.

Reply:
That should be an error anyway. If you make it an error, does any test or CI fail?

Reply:
It should be error anyway. My bad. Does any test/CI depend on this behavior?